src/stories/smart-components/oe-actor-widget.stories.ts

```diff
@@ -55,13 +55,13 @@ export const Default: Story = {
     setup() {
       const actorType = ActorType.PERSOON;
       const id = '1';
-      const api = 'https://dev-actoren.onroerenderfgoed.be';
+      const api = 'http://local.onroerenderfgoed.be:6543/';
       const getSsoToken = async () => 1;
```

Comment:
Kunnen we deze niet aanpassen dat er geen ssotoken meegegeven wordt in de call naar actoren, als die leeg is? op zich is die api deels publiek, dus geeft die sowieso antwoordt.